### ISSUE TITLE
the content of consent form is hard-coded in the `sign` component

### ISSUE THREAD

**Issue by yibeichan** [ReproNim/reproschema-ui#319]
Hi @satra,
I tried to add a consent sign in the demo project for the reproschema paper but then noticed that the content in `sign` in UI was hard-coded as follows:
https://github.com/ReproNim/reproschema-ui/blob/2129a41e2238066f5c32bb076ee00e382398cc39/src/components/StudySign/StudySign.vue#L21-L24
so the consent form will always look like http://www.repronim.org/reproschema-demo/#/activities/0

do you know why it is designed this way? Shouldn't it be that `sign` is only a function/interface that allows a participant to sign their name and then click "agree/disagree" and the content of the consent form can be defined separately in each research protocol (in the current case, our demo activity here https://github.com/ReproNim/reproschema-demo/tree/main/activities/consent)

**Comment by satra** [comment]
yes, that's how it should be.  however, i think we stopped working on it because we could not use it for consent. mit requested that we use docusign or redcap.

**Comment by yibeichan** [comment]
make sense; but wouldn't it be better if we update it since we want researchers beyond MIT to use it? It's always better to have the option of (not) using something than to have no option? If yes, then I can work on it.

**Comment by satra** [comment]
sure, but i would prioritize all the other things before this one.

**Comment by yibeichan** [comment]
sounds good; l'll finish all other things first
